### DISCUSSION TITLE
Relationship  how to prevent sort after save operation?

### DISCUSSION THREAD

**Discussion by marselo777** [keystonejs/keystone#8761] [Questions] [answered]
Hi! I'm using the `relationship` field in `select mode`, and when I add a new value to select, it looks well, but `after save` operation, I see that my added values to select were sorted, how can I prevent this behaviour? I want to see the same positions of items that I select.

**Comment by dcousens** [comment]
@marselo777 can you show a video of what behaviour you are talking about, including when and what it happens?

**Reply by marselo777** [reply]
@dcousens Hi! Thanks for respond,
Here is an example [video](https://github.com/keystonejs/keystone/assets/65085710/e4151c8b-06a6-4407-9c64-9f03f4966a68) to show that I expect 

**What I expect:** 
As you can see I've saved it like this 
![telegram-cloud-photo-size-2-5391216716182309511-y](https://github.com/keystonejs/keystone/assets/65085710/d082b3f1-ddf9-4051-8f37-15253a132f7e)
But after save, my order of items in select was changed to  this: 
<img width="743" alt="Screenshot 2023-08-14 at 13 10 10" src="https://github.com/keystonejs/keystone/assets/65085710/9daa9a38-7e7f-4529-b3d1-a6cdd766f776">

**Reply by dcousens** [reply]
Ah, I don't think we can resolve that, relationships don't have an order in the database.

**Reply by dcousens** [reply, marked as answer]
https://github.com/keystonejs/keystone/pull/7352 might be required to help you resolve this if you could define a sort order, potentially using another field which records when relationships were connected.

**Reply by marselo777** [reply]
@dcousens , Thanks for answer, will try to use the info from ticket, that you've attached. Have a nice day! 👍
